docs/content/advanced/backend/upgrade-guides/1-7-1.md

```diff
@@ -112,4 +112,4 @@ type PaymentSessionResponse = {
 }
 ```
 
-Where `session_data` would include the previously returned data from these methods.
+Where `session_data` would include the previously returned data from these methods. `update_requests` allows you to request from the core to update data of internal resources. Currently, it can only be used to update the `metadata` field of the customer entity.
```

Comment:
suggestion: Sorry for nitpicking!

_The property `update_requests` allows you to pass data from the payment provider plugin to the core to update internal resources. Currently, it can only be used to update the `metadata` field of the customer entity._ 

And for the documentation, I think it'd be good to show the Stripe example